packages/react-dom-bindings/src/client/ReactDOMComponentTree.js

```diff
@@ -140,15 +148,15 @@ export function getClosestInstanceFromNode(targetNode: Node): null | Fiber {
           // Let's get the fiber associated with the SuspenseComponent
           // as the deepest instance.
           // $FlowFixMe[prop-missing]
-          const targetSuspenseInst = suspenseInstance[internalInstanceKey];
-          if (targetSuspenseInst) {
-            return targetSuspenseInst;
+          const targetFiber = hydrationInstance[internalInstanceKey];
+          if (targetFiber) {
+            return targetFiber;
           }
```

Comment:
>_possible bug_

**✨ AI Suggestion:** The `targetFiber` variable is not guaranteed to be defined. Add a null check before returning it.
```suggestion
          const targetFiber = hydrationInstance[internalInstanceKey];
          if (targetFiber !== null) {
            return targetFiber;
          }
```